tests/cupyx_tests/scipy_tests/sparse_tests/test_construct.py

```diff
@@ -252,43 +252,43 @@ def test_random(self):
         x = getattr(sparse, self.random_method)(
             3, 4, density=0.1,
             format=self.format, dtype=self.dtype)
-        self.assertEqual(x.shape, (3, 4))
-        self.assertEqual(x.dtype, self.dtype)
-        self.assertEqual(x.format, self.format)
+        assert x.shape == (3, 4)
+        assert x.dtype == self.dtype
+        assert x.format == self.format
 
     def test_random_with_seed(self):
         x = getattr(sparse, self.random_method)(
             3, 4, density=0.1,
             format=self.format, dtype=self.dtype,
             random_state=1)
-        self.assertEqual(x.shape, (3, 4))
-        self.assertEqual(x.dtype, self.dtype)
-        self.assertEqual(x.format, self.format)
+        assert x.shape == (3, 4)
+        assert x.dtype == self.dtype
+        assert x.format == self.format
 
         y = getattr(sparse, self.random_method)(
             3, 4, density=0.1,
             format=self.format, dtype=self.dtype,
             random_state=1)
 
-        self.assertTrue((x.toarray() == y.toarray()).all())
+        assert (x.toarray() == y.toarray()).all()
```

Comment:
How about using `testing.assert_array_equal`?
```suggestion
        testing.assert_array_equal(x.toarray(), y.toarray())
```